src/main/resources/aws/2-nodes-dc.yaml

```diff
@@ -36,6 +36,7 @@ Resources:
       KeyName:
         Ref: KeyName
       ImageId: !Ref Ami
+      InstanceInitiatedShutdownBehavior: terminate
```

Comment:
This behavior is significant. We want to rely on it externally. Therefore we want it to be a behavioral contract (KeepAChangelog `Added`, SemVer MINOR). [We specify behavioral contracts via Javadoc](https://github.com/atlassian/aws-infrastructure/commit/568f949ca1f4f4ec5f6592adab42592ba094061f#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12). So let's add this behavior to:
* `DataCenterFormula`
* `StandaloneFormula`
* `StackVirtualUsersFormula`

While we think of the public API, let's not forget about its other parts, not covered yet:
* `Ec2VirtualUsersFormula`
* `ApacheEc2LoadBalancerFormula`

Also, tests are still "vulnerable": `AwsCliIT`
The common point for the non-Cloudformation ones is `aws.awaitingEc2.allocateInstance`, perhaps we could put the terminate as default there.

Reply:
aws.awaitingEc2.allocateInstance - maybe we should make the change in both places?

Reply:
i.e. here and in aws-resource